app/about/data-extraction/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
```

Comment:
These are intentionally empty for now, this is all behind a feature flag so I'm not making us look bad hahaha